data/strings/types_strings.txt

```diff
@@ -27694,6 +27713,8 @@
 
   [type.waterway.ditch.tunnel]
     ref = type.waterway.ditch
+    en = Culvert
```

Comment:
Good! Can you please add the tunnel translations for other languages too using [Wiki](https://en.wikipedia.org/wiki/Culvert) and other translation services?

Reply:
Done,
Some languages are not available in Wikipedia, i have check translation on DeepL but satisfying.
I wait translations are approved to regenerate strings